pom.xml

```diff
@@ -141,6 +142,7 @@
                                         <source>${project.basedir}/src/main/301until310-nondb/scala</source>
                                         <source>${project.basedir}/src/main/301until320-nondb/scala</source>
                                         <source>${project.basedir}/src/main/301until320-all/scala</source>
+                                        <source>${project.basedir}/src/main/301until310-all/scala</source>
```

Comment:
nit: There are getting to be enough directories now, that it would be good to have a consistent ordering to them, so it is simpler to tell if something is missing for example.

Reply:
ah, I missed a few I tried moved some of them but didn't get them all. I need to update my IntelliJ as well as Gera mentioned it has a sort lines in new version, but we should decide on order

Reply:
sorted them